strapi/strapi
# enh: add logger level to server config

### What does it do?

Loads log level from server.logger.config

### Why is it needed?

There was no documented way to change the log level, and loading it as a string from config/logger.js was not ideal

### How to test it?

Set server.logger.config or logger and it will use those values to pass to the logger setup for the log level

Default when nothing is set should be 'info'

### Related issue(s)/PR(s)

Let us know if this is related to any issue/pull request
